docs/Privacy/Start-Pantheon-Privacy.md

```diff
@@ -0,0 +1,334 @@
+
+#### rpc-http-api
+
+```bash tab="Example Command Line"
+--rpc-http-api=EEA
+```
+
+```bash tab="Example Configuration File"
+rpc-http-api=["EEA"]
+```
+
+Comma-separated APIs to enable on the HTTP JSON-RPC channel.
+When you use this option, the `--rpc-http-enabled` option must also be specified.
+The available API options are: `ADMIN`, `ETH`, `NET`, `WEB3`, `CLIQUE`, `IBFT`, `PERM`, `DEBUG`, `MINER`, and `EEA`.
+The default is: `ETH`, `NET`, `WEB3`.
+
+!!!note
+    EEA methods are for privacy features. Privacy features are under development and will be available in v1.1.  
+
+!!!tip
+    The singular `--rpc-http-api` and plural `--rpc-http-apis` are available and are just two
+    names for the same option.
+
+#### privacy-enabled
+
+```bash tab="Example Command Line"
+--privacy-enabled=true
+```
+
+```bash tab="Example Configuration File"
+privacy-enabled=true
+```
+
+Set to enable private transactions. 
+The default is false.
+
+!!!note
+    Privacy is under development and will be available in v1.1.  
+
+#### privacy-precompiled-address
+
+```bash tab="Example Command Line"
+--privacy-precompiled-address=125
+```
+
+```bash tab="Example Configuration File"
+privacy-precompiled-address=125
+```
+
+Address to which the privacy pre-compiled contract is mapped.
+The default is 126. 
+
+!!!note
+    Privacy is under development and will be available in v1.1.    
+
+
+### Enclave (Orion)
+
+#### privacy-public-key-file
+
+```bash tab="Syntax"
+--privacy-public-key-file=<privacyPublicKeyFile>
+```
+
+Path to the public key for the enclave.     
+
+!!!note
+    Privacy is under development and will be available in v1.1.
+
+!!!note
+    This option is not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#privacy-public-key-file).
+
+#### privacy-url
+
+```bash tab="Syntax"
+--privacy-url=<privacyUrl>
+```
+
+URL on which enclave is running.    
+
+!!!note
+    Privacy is under development and will be available in v1.1.
+    
+    
+### Privacy JSON-RPC API method
+
+The [EEA methods](../Reference/JSON-RPC-API-Methods.md#eea_sendRawTransaction) created to
+provide and support privacy.
+
+### Set Privacy Up
+
+## Prerequisites 
+
+[Pantheon](../Installation/Install-Binaries.md) 
+
+[Curl (or similar web service client)](https://curl.haxx.se/download.html) 
+
+## Steps
+
+To create a private network: 
+
+1. [Create Folders](#1-create-folders)
+1. [Create Genesis File](#2-create-genesis-file)
+1. [Get Public Key of First Node](#3-get-public-key-of-first-node)
+1. [Start First Node as Bootnode](#4-restart-first-node-as-bootnode)
+1. [Start Node-2](#5-start-node-2)
+6. [Start Node-3-non-privacy](#6-start-node-3)
+7. [Confirm the private is working](#7-confirm-private-network-is-working)
+8. [Create a Private Transaction Node-1 with Node-2](#8-create-a-private-transaction-node-1-with-node-2)
+9. [Confirm Node-3 can't interact with private Transaction](#9-confirm-node-3-can't-interact-with-private-transaction)
+
```

Comment:
Don't we need steps in here to start Orion nodes for each Pantheon node?

Reply:
sorted